File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    let even = 0;
    let odd = 0;
    for(let i =0; i<num_list.length; i++){
        if(num_list[i] % 2 === 0){
            odd++;
        }else{
            even++;
        }
    }
    
    return [odd, even];
}

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[2];
        int 홀수 = 0;
        int 짝수 = 0;
        
        for(int num : num_list){
            if(num % 2 == 0){
                짝수++;
            }else{
                홀수++;
            }
        }
        
        answer[0] = 짝수;
        answer[1] = 홀수;
        
        return answer;
    }
}